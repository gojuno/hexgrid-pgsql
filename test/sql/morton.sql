
select morton_pack(-13, 42) as code;

select morton_unpack(morton_pack(-13, 42)) as hex_qr;
