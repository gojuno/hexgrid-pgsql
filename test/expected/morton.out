select morton_pack(-13, 42) as code;
        code         
---------------------
 4611686018427390169
(1 row)

select morton_unpack(morton_pack(-13, 42)) as hex_qr;
  hex_qr  
----------
 {-13,42}
(1 row)

